packages/ui/src/components/common/TransactionModal/TransactionModalRenderer.tsx

```diff
@@ -274,7 +274,7 @@ export const TransactionModalRenderer: FC<Props> = ({
             is_canonical: useExternalLiquidity,
             txHashes: steps
               ?.map((step) => {
-                let txHashes: { chainId: number; txHash: Address }[] = []
+                let txHashes: { chainId: number; txHash: string }[] = []
```

Comment:
is this in preparation for solana? If so we should just add a proper type of Address or solana tx hash.